rsocket-core/src/main/java/io/rsocket/resume/BufferStrategy.java

```diff
@@ -0,0 +1,15 @@
+package io.rsocket.resume;
+
+import io.rsocket.Frame;
+
+public interface BufferStrategy {
```

Comment:
This name isn't great. Can this be called something like `ResumePositionCounter`?. It's not really something that does buffering.